core/modules/node/node.module

```diff
@@ -1148,7 +1148,66 @@ function node_view(Node $node, $view_mode = 'full', $langcode = NULL) {
  *   content language of the current request.
  */
 function node_build_content(Node $node, $view_mode = 'full', $langcode = NULL) {
-  $node->buildContent($view_mode, $langcode);
+  if (!isset($langcode)) {
```

Comment:
This whole section is mostly reverting a change where we moved the logic from here to `$node->buildContent()`, where it should be. This should not be moved back here.